Dockerfile

```diff
@@ -1,18 +1,22 @@
-FROM phusion/baseimage:0.9.18
+FROM python:3.6-slim-stretch
 MAINTAINER Holger Schinzel <holger@dash.org>
 
-ARG USER_ID
-ARG GROUP_ID
+RUN apt-get update && \
+    apt-get install git curl mysql-client cython3 -y && \
+    apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+ENV SENTINEL_COMMIT=01ab29702cebb68503dac6f1cda5d33f327ca492
```

Comment:
This could be a build arg instead of hard-coded in the Dockerfile

Reply:
Changed to ARG